modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -59,23 +60,47 @@ You can use the following example to match the `pipeline-pr-main` pipeline with
 [discrete]
 .Matching a push event to a pipeline run
 
-You can use the following example to match the `pipeline-push-on-main` pipeline with a `push` event targeting the `refs/heads/main` branch:
+You can use the following example to match the `pipeline-push-on-main` pipeline run with a `push` event targeting the `refs/heads/main` branch:
 
 [source,yaml]
 ----
-...
-  metadata:
-    name: pipeline-push-on-main
-  annotations:
-    pipelinesascode.tekton.dev/on-target-branch: "[refs/heads/main]" <1>
-    pipelinesascode.tekton.dev/on-event: "[push]"
-...
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-push-on-main
+annotations:
+  pipelinesascode.tekton.dev/on-target-branch: "[refs/heads/main]" <1>
+  pipelinesascode.tekton.dev/on-event: "[push]"
+# ...
 ----
-<1> You can specifiy multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following:
+<1> You can specify multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following:
 * Full references to branches such as `"refs/heads/main"`
 * Globs with pattern matching such as `"refs/heads/\*"`
 * Tags such as `"refs/tags/1.\*"`
 
+[discrete]
+.Matching a comment event to a pipeline run
+
+You can use the following example to match the `pipeline-comment` pipeline run with a comment on a pull request, when the text of the comment matches the `^/merge-pr` regular expression:
```

Comment:
@chmouel we also support `on-comment` for push as well right ?

Reply:
@savitaashture no still waiting this to be merged 🙃 https://github.com/openshift-pipelines/pipelines-as-code/pull/1638 (would not make it for osp1.5 now as there is not enough time to dogfood this)

Reply:
Ah i see